ccw.core.test/Counterclockwise Product Tests.launch

```diff
@@ -28,17 +32,563 @@
+<mapEntry key="org.eclipse.wst.internet.monitor.core/config" value="false"/>
+<mapEntry key="org.eclipse.wst.internet.monitor.core/debug" value="true"/>
+<mapEntry key="org.eclipse.wst.internet.monitor.core/finest" value="false"/>
+<mapEntry key="org.eclipse.wst.internet.monitor.core/parsing" value="false"/>
+<mapEntry key="org.eclipse.wst.internet.monitor.core/severe" value="false"/>
+<mapEntry key="org.eclipse.wst.internet.monitor.core/warning" value="false"/>
+<mapEntry key="org.eclipse.wst.server.core/config" value="false"/>
+<mapEntry key="org.eclipse.wst.server.core/debug" value="true"/>
+<mapEntry key="org.eclipse.wst.server.core/extension_point" value="false"/>
+<mapEntry key="org.eclipse.wst.server.core/finer" value="false"/>
+<mapEntry key="org.eclipse.wst.server.core/finest" value="false"/>
+<mapEntry key="org.eclipse.wst.server.core/info" value="false"/>
+<mapEntry key="org.eclipse.wst.server.core/listeners" value="false"/>
+<mapEntry key="org.eclipse.wst.server.core/performance" value="false"/>
+<mapEntry key="org.eclipse.wst.server.core/publishing" value="false"/>
+<mapEntry key="org.eclipse.wst.server.core/resources" value="false"/>
+<mapEntry key="org.eclipse.wst.server.core/runtime_target" value="false"/>
+<mapEntry key="org.eclipse.wst.server.core/severe" value="false"/>
+<mapEntry key="org.eclipse.wst.server.core/warning" value="false"/>
+<mapEntry key="org.eclipse.wst.server.discovery/debug" value="true"/>
+<mapEntry key="org.eclipse.wst.server.ui/config" value="false"/>
+<mapEntry key="org.eclipse.wst.server.ui/debug" value="true"/>
+<mapEntry key="org.eclipse.wst.server.ui/extension_point" value="false"/>
+<mapEntry key="org.eclipse.wst.server.ui/finer" value="false"/>
+<mapEntry key="org.eclipse.wst.server.ui/finest" value="false"/>
+<mapEntry key="org.eclipse.wst.server.ui/info" value="false"/>
+<mapEntry key="org.eclipse.wst.server.ui/performance" value="false"/>
+<mapEntry key="org.eclipse.wst.server.ui/severe" value="false"/>
+<mapEntry key="org.eclipse.wst.server.ui/warning" value="false"/>
+<mapEntry key="org.eclipse.wst.sse.core/debug" value="true"/>
+<mapEntry key="org.eclipse.wst.sse.core/dom/adapter/notification/time" value="false"/>
+<mapEntry key="org.eclipse.wst.sse.core/dom/adapter/notification/time/criteria" value="10"/>
+<mapEntry key="org.eclipse.wst.sse.core/filebuffers/leaks" value="true"/>
+<mapEntry key="org.eclipse.wst.sse.core/filebuffers/lifecycle" value="false"/>
+<mapEntry key="org.eclipse.wst.sse.core/filebuffers/modelmanagement" value="false"/>
+<mapEntry key="org.eclipse.wst.sse.core/format" value="false"/>
+<mapEntry key="org.eclipse.wst.sse.core/structureddocument" value="false"/>
+<mapEntry key="org.eclipse.wst.sse.core/structuredmodel/lifecycle" value="false"/>
+<mapEntry key="org.eclipse.wst.sse.core/structuredmodel/locks" value="true"/>
+<mapEntry key="org.eclipse.wst.sse.core/structuredmodel/modelmanager" value="false"/>
+<mapEntr
```

Comment:
Is this super extra big addition to the file necessary? Won't it be fragile if used on different environments?